src/main/kotlin/org/jitsi/jibri/selenium/status_checks/EmptyCallStatusCheck.kt

```diff
@@ -44,6 +46,13 @@ class EmptyCallStatusCheck(
     private fun CallPage.isCallEmpty() = getNumParticipants() <= 1
 
     companion object {
-        val DEFAULT_CALL_EMPTY_TIMEOUT: Duration = Duration.ofSeconds(30)
+        private val defaultCallEmptyTimeout: Duration by config {
+            "JibriConfig::defaultCallEmptyTimeout" {
+                Duration.ofSeconds(Config.legacyConfigSource.defaultCallEmptyTimeout!!)
+            }
+            "jibri.default-call-empty-timeout".from(Config.configSource).convertFrom<Long>(Duration::ofSeconds)
```

Comment:
Let's just have this parse as a `Duration` directly from `reference.conf`.  Also, let's scope these under a common object.  So you can set this property there like this:
```
jibri {
   call-status-checks {
       call-empty-timeout = 30 seconds
   }   
}
```